Compentio.SourceMapper.Generator/Generators/MainSourceGenerator.cs

```diff
@@ -27,7 +27,7 @@ public void Execute(GeneratorExecutionContext context)
             if (context.SyntaxReceiver is not MappersSyntaxReceiver receiver)
                 return;
 
-            var sourcesMetadata = SourcesMetadata.Create(context.Compilation.ReferencedAssemblyNames);
+            var sourcesMetadata = SourcesMetadata.Create(context.Compilation.ReferencedAssemblyNames, context.Compilation.SourceModule.ReferencedAssemblySymbols);
```

Comment:
Redundancy: it can be achieved with one parameter: `context.Compilation` in create.

Reply:
Fixed.